app/src/main/java/nextstep/jwp/http/HttpRequest.java

```diff
@@ -0,0 +1,77 @@
+package nextstep.jwp.http;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class HttpRequest {
+    private final RequestLine requestLine;
+    private final Map<String, String> headers = new HashMap<>();
+    private final Map<String, String> params = new HashMap<>();
+    private final BufferedReader bufferedReader;
+
+    public HttpRequest(InputStream inputStream) throws IOException {
+        bufferedReader = new BufferedReader(new InputStreamReader(inputStream));
+        requestLine = new RequestLine(bufferedReader);
+        parseHeaders();
+        parseParams();
+    }
+
+    private void parseHeaders() throws IOException {
+        while (bufferedReader.ready()) {
+            String line = bufferedReader.readLine();
+            if (Objects.isNull(line) || line.isEmpty()) {
```

Comment:
아래와 같이 변경해볼 수도 있을 것 같아요. 😃 
```suggestion
            if (Strings.isNullOrEmpty(line)) {
```

Reply:
오호 좋은거 알아갑니다!